File: Python/2708.py
entrada = input()
jeepSituation = entrada.split()[0]
if len(entrada.split()) > 1:
    t = int(entrada.split()[1])

qntJeepSalida, qntJeepVuelta = 0, 0
tSalida, tVuelta = 0, 0

while jeepSituation != "ABEND":
    if jeepSituation == "SALIDA":
        qntJeepSalida += 1
        tSalida += t
    else:
        qntJeepVuelta += 1
        tVuelta += t

    entrada = input()
    jeepSituation = entrada.split()[0]
    if len(entrada.split()) > 1:
        t = int(entrada.split()[1])

print(tSalida - tVuelta)
print(qntJeepSalida - qntJeepVuelta)

File: C/2708.c
#include <stdio.h>
#include <string.h>

int main(){

    char jeepSituation[7];
    int t, tSalida = 0, tVuelta = 0, qntJeepSalida = 0, qntJeepVuelta = 0;

    scanf("%s", jeepSituation);

    while(strcmp(jeepSituation, "ABEND") != 0){
        scanf("%i", &t);

        if (strcmp(jeepSituation, "SALIDA") == 0){
            qntJeepSalida++;
            tSalida += t;
        }else{
            qntJeepVuelta++;
            tVuelta += t;
        }

        scanf("%s", jeepSituation);
    }

    printf("%i\n%i\n", (tSalida - tVuelta), (qntJeepSalida - qntJeepVuelta));

    return 0;
}